homeassistant/components/ads/sensor.py

```diff
@@ -17,13 +17,16 @@
     {
         vol.Required(CONF_ADS_VAR): cv.string,
         vol.Optional(CONF_ADS_FACTOR): cv.positive_int,
+        vol.Optional(CONF_ADS_DIGIT, default=1): cv.positive_int,
```

Comment:
Please break out magic integers and strings as constants.

Reply:
fixed.